functions/src/utils/user.js

```diff
@@ -0,0 +1,17 @@
+const admin = require("firebase-admin");
+
+/** Add a user to the list of admins
+ *
+ * @param {string} newAdminId
+ */
+const addAdmin = async (newAdminId) => {
```

Comment:
[NB] moved `addAdmin` to separate utility file as to not create direct dependency between `auth.js` and `venue.js` files that both `require()` it

Reply:
Please re-read my original request in https://github.com/sparkletown/sparkle/pull/1829#discussion_r668750203 about moving both `addAdmin` and `removeAdmin` into `api/roles`

Reply:
moved 3b1e532e09e585f60dd5b0c919666db539bb8f73